custom-metrics-stackdriver-adapter/README.md

```diff
@@ -94,14 +99,23 @@ in Prometheus format.
 The name of your metric must start with custom.googleapis.com/ prefix followed
 by a simple name, as defined in [custom metric naming rules].
 
+You will report your metric against a proper monitored resource for Kubernetes
```

Comment:
When you say proper do you mean appropriate?

Reply:
Yes

Reply:
Sorry for picking on this, looks like I had a wrong understanding of what proper actually means in English :)